apps/wakunode2/app.nim

```diff
@@ -547,7 +547,16 @@ proc setupProtocols(node: WakuNode, conf: WakuNodeConf,
     peerExchangeHandler = some(handlePeerExchange)
 
   if conf.relay:
-    let pubsubTopics = conf.topics.split(" ")
+
+    var pubsubTopics = @[""]
+    if conf.topicsDeprecated != "":
```

Comment:
How about logging a warning if `conf.topicsDeprecated != ""` ?

Reply:
Good point! I'll add it

Reply:
agree.

```
if conf.topicsDeprecated != "" and conf.topics.len !=0.
  error

if conf.topicsDeprecated != "":
  warn
```

Reply:
Now, if `topicsDeprecated` is given, it prints a `warn`. If both `topicsDeprecated` and `topics` are given, then an error is returned and the execution is halted.